functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/customAssertions/BuildProjectAssert.java

```diff
@@ -0,0 +1,59 @@
+package org.arquillian.smart.testing.ftest.customAssertions;
+
+import java.io.File;
+import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.api.FileAssert;
+import org.jboss.shrinkwrap.resolver.api.maven.embedded.BuiltProject;
+
+public class BuildProjectAssert extends AbstractAssert<BuildProjectAssert, BuiltProject> {
+
+    BuildProjectAssert(BuiltProject actual) {
+        super(actual, BuildProjectAssert.class);
+    }
+
+    public static BuildProjectAssert assertThat(BuiltProject module) {
+        return new BuildProjectAssert(module);
+    }
+
+    public BuildProjectAssert hasAllBuiltSubModulesContainEffectivePom(String effectivePom) {
```

Comment:
How about just `allSubModulesContainEffectivePom`
analogically for the following method